packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -56,6 +56,7 @@ export class ZipkinExporter implements SpanExporter {
         method: 'POST',
         headers: {
           'Content-Type': 'application/json',
+          'x-ot-request': 1,
```

Comment:
Could you please create const for header name and `s/x-ot-request/x-opentelemetry-outgoing-request`?

Reply:
Do you mean create a const in opentelemetry-types package or just in the ZipkinExporter class ?